Website/package.json

```diff
@@ -15,10 +15,12 @@
     "react-chartjs-2": "^2.9.0",
     "react-datetime": "2.16.3",
     "react-dom": "16.13.1",
+    "react-map-gl": "^5.2.3",
```

Comment:
Cannot see any code changes that uses this dependencies, please remove and only include when you finish the map work.